src/lib.rs

```diff
@@ -70,7 +70,7 @@
 //! bo.write(&buffer).unwrap();
 //!
 //! // create a framebuffer from our buffer
-//! let fb = gbm.add_framebuffer(&bo).unwrap();
+//! let fb = gbm.add_framebuffer(&bo, 1, 32).unwrap();
```

Comment:
Please use `, 32, 32` as arguments here

Reply:
We need just append variable assignment line at end of `$GITHUB_ENV` file.
The `File::create` truncates existing file, which is wrong usage.

Reply:
Weird, me neither. In that case, the File::create discussion is also resolved for me.

Reply:
> This reply addressed to comment above, seems I can't reply to it.

That is because [that comment](https://github.com/Smithay/gbm.rs/pull/4#discussion_r624532695) was sent in reply to [my comment](https://github.com/Smithay/gbm.rs/pull/4#discussion_r612706392) in [my review](https://github.com/Smithay/gbm.rs/pull/4#pullrequestreview-634925648), as part of [Drakulix' review](https://github.com/Smithay/gbm.rs/pull/4#pullrequestreview-649800089). GitHub does not really know what to do in that case, and simply renders it twice. Once in Drakulix' review without reply field, the second time as part of the original "comment thread". You can easily jump to the original by clicking the "xxx hours ago", that gives a unique permalink to the comment-in-thread (where you can reply) for both of the rendered instances.